packages/sandbox/src/app/sandbox/docs/consent-box.doc.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, ConsentBox } from '@getluko/streamline';
+import React, { useState } from 'react';
+
+export const ConsentBoxSandbox = () => {
+  const [checked, setChecked] = useState(false);
+  const [dangerChecked, setDangerChecked] = useState(false);
+  return (
+    <Box margin="xs">
+      <Box margin="xs">
+        <ConsentBox
+          value={checked}
+          onChange={(v) => setChecked(v)}
+          text="Primary"
+        />
+      </Box>
+      <Box margin="xs">
+        <ConsentBox
+          appearance="danger"
+          value={dangerChecked}
+          onChange={(v) => setDangerChecked(v)}
+          text="Danger"
+        />
+      </Box>
+      <Box margin="xs">
+        <ConsentBox
+          appearance="primary"
+          isDisabled
+          value={checked}
+          onChange={(v) => setDangerChecked(v)}
+          text="Primary disabled"
+        />
+      </Box>
+      <Box margin="xs">
+        <ConsentBox
+          appearance="danger"
+          isDisabled
+          value={dangerChecked}
+          onChange={(v) => setDangerChecked(v)}
+          text="Danger disabled"
```

Comment:
@hcourthias Can you add a test case when the text is on several lines to make sure the checkbox alignment and text cropping works accordingly?